Add events ModifyInfoModuleContentEvent, ProvideStaticVariablesToComponentEvent and RenderComponentEvent to event list (#6465)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Fluid/UsingFluidInTypo3.rst

defines a component that already exists in `vendor_extension`, it will override
the original component in `vendor_extension`.

.. _psr-14-events_fluid_components:

PSR-14 events related to Fluid components
-----------------------------------------

> Three PSR-14 events have been added to influence the processing and rendering

Reviewer: ```suggestion
Three PSR-14 events are available to influence the processing and rendering
```
